src/modules/fetchComment.js

```diff
@@ -0,0 +1,37 @@
+import { postComments, getComments } from './commentApi.js';
+
+const sendComment = async (id) => {
+  const submitBtn = document.querySelector(`#item${id}`);
+  submitBtn.addEventListener('click', async (e) => {
+    const username = document.querySelector('.username').value;
+    const usercomment = document.querySelector('.usercomment').value;
+    const form = document.querySelector('.Form');
+    if (username.length !== 0 && usercomment.length !== 0) {
+      await postComments(`item${id}`, username, usercomment);
+      form.reset();
+      e.preventDefault();
+    }
+  });
+};
+const getAllComment = async (id) => {
+  // const container = document.getElementById(`${id}`);
+  const table = document.querySelector('.table');
+  // const submitBtn = document.querySelector(`#item${id}`);
```

Comment:
Kindly, take out unused code lines. This will make your codebase clean, readable and dry.